src/styles/pages/_pagination.scss

```diff
@@ -0,0 +1,46 @@
+@import '../settings/colors';
+
+$W_PAGINATION_LINK_SPACING: 6px;
+
+@mixin w-pagination__links($COLOR, $ACTIVE: false) {
+  display: flex;
+  font-size: 20px;
+  justify-content: center;
+  min-width: 20px;
+  padding: 0px $W_PAGINATION_LINK_SPACING;
+
+  a {
+    color: $COLOR;
+    text-decoration: none;
+
+    @if $ACTIVE {
+      border-bottom: 2px solid $COLOR;
+    } @else {
+      &:active,
+      &:focus,
+      &:hover {
+        border-bottom: 2px solid $COLOR;
+      }
+    }
+  }
+}
+
+.w-pagination {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+}
+
+.w-pagination > .w-pagination__active {
```

Comment:
I think the BEM style is that modifiers should have two dashes instead of underscores. So it would be:

`.w-pagination__link .w-pagination--active`

I realize that kind of blows up the mixin though. IMO it may be good to separate the modifier from the link styles in case we decide to add more modifiers in the future. It's also good to avoid unnecessary selector specificity if we can help it. So removing the `>` would be good too.